### ISSUE TITLE
Modified videos are not reloaded upon subsequent opens

### ISSUE THREAD

**Issue by gkertai** [videovillage/Screen-Issues#28]
Once you have a video loaded in Screen, it will not reflect changes to the file, even if you open the file again (without closing it first).

Typical use case: I have Screen set to the default video player. I export an animation from Apple Motion, which automatically opens it in Screen. If I modify the animation and overwrite the same file name, Motion opens the file again in Screen, but Screen will still play back the previous version of the file (unless I close it then reopen manually).

Looks like there's some internal cache that would need to be invalidated when the original file changes.

**Comment by gregcotten** [comment]
Yes, Screen definitely does not reload videos. When you try to open a video that's already open, Screen just brings that window to the front and doesn't perform a reload. 

This all depends on the video and underlying caching, but generally if you have a file open the system will essentially "lock" the file and even if you write over it, the app that already has the file loaded will still have a copy of the original file. That original file disappears when the lock is lifted.

What Screen **should** have is a reload button. I'll get on that!

**Comment by gkertai** [comment]
A reload button sounds nice. Even a File menu item with a keyboard shortcut would do (apparently ⌘R is already taken, but ⌥⌘R seems available).

What users would really benefit from is a *notification* that the file has changed and the reload button shall be used at their discretion. Kind of like the existing ![Screen Shot 2020-03-04 at 22 36 01](https://user-images.githubusercontent.com/1874498/75925193-90f7f380-5e68-11ea-91f9-fdb2b3544966.PNG) indicator, which is only visible when relevant.
